molo/core/models.py

```diff
@@ -1532,6 +1535,12 @@ def is_commenting_enabled(self):
             return False
         return True
 
+    def get_anonymous_commenting_alias(self):
```

Comment:
I don't think this method is needed if blank is false and the default is provided